tools/tuf/README.md

```diff
@@ -0,0 +1,292 @@
+TARGETS_PASSPHRASE_1PASSWORD_PATH="Private/TUF TARGETS/password" \
+SNAPSHOT_PASSPHRASE_1PASSWORD_PATH="Private/TUF SNAPSHOT/password" \
+TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/TUF TIMESTAMP/password" \
+GITHUB_USERNAME=foobar \
+GITHUB_TOKEN_1PASSWORD_PATH="Private/Github Token/password" \
+PUSH_TO_REMOTE=1 \
+./tools/tuf/releaser.sh
+```
+
+#### Releasing osquery `5.12.1` to `edge`
+
+```sh
+AWS_PROFILE=tuf \
+TUF_DIRECTORY=/Users/luk/tuf.fleetctl.com \
+COMPONENT=osqueryd \
+ACTION=release-to-edge \
+VERSION=5.12.0 \
+KEYS_SOURCE_DIRECTORY=/Volumes/FLEET-TUF/keys \
+TARGETS_PASSPHRASE_1PASSWORD_PATH="Private/TUF TARGETS/password" \
+SNAPSHOT_PASSPHRASE_1PASSWORD_PATH="Private/TUF SNAPSHOT/password" \
+TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/TUF TIMESTAMP/password" \
+GITHUB_USERNAME=foobar \
+GITHUB_TOKEN_1PASSWORD_PATH="Private/Github Token/password" \
+PUSH_TO_REMOTE=1 \
+./tools/tuf/releaser.sh
+```
+
+### Promoting from `edge` to `stable`
+
+#### Promoting fleetd `1.23.0` from `edge` to `stable`
+
+```sh
+AWS_PROFILE=tuf \
+TUF_DIRECTORY=/Users/foobar/tuf.fleetctl.com \
+COMPONENT=fleetd \
+ACTION=promote-edge-to-stable \
+VERSION=1.23.0 \
+KEYS_SOURCE_DIRECTORY=/Volumes/FLEET-TUF/keys \
+TARGETS_PASSPHRASE_1PASSWORD_PATH="Private/TUF TARGETS/password" \
+SNAPSHOT_PASSPHRASE_1PASSWORD_PATH="Private/TUF SNAPSHOT/password" \
+TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/TUF TIMESTAMP/password" \
+GITHUB_USERNAME=foobar \
+GITHUB_TOKEN_1PASSWORD_PATH="Private/Github Token/password" \
+PUSH_TO_REMOTE=1 \
+./tools/tuf/releaser.sh
+```
+
+#### Promoting osqueryd `5.12.1` from `edge` to `stable`
+
+```sh
+AWS_PROFILE=tuf \
+TUF_DIRECTORY=/Users/foobar/tuf.fleetctl.com \
+COMPONENT=osqueryd \
+ACTION=promote-edge-to-stable \
+VERSION=5.12.0 \
+KEYS_SOURCE_DIRECTORY=/Volumes/FLEET-TUF/keys \
+TARGETS_PASSPHRASE_1PASSWORD_PATH="Private/TUF TARGETS/password" \
+SNAPSHOT_PASSPHRASE_1PASSWORD_PATH="Private/TUF SNAPSHOT/password" \
+TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/TUF TIMESTAMP/password" \
+GITHUB_USERNAME=foobar \
+GITHUB_TOKEN_1PASSWORD_PATH="Private/Github Token/password" \
+PUSH_TO_REMOTE=1 \
+./tools/tuf/releaser.sh
+```
+
+#### Releasing `swiftDialog` to `stable`
+
+> `releaser.sh` doesn't support `swiftDialog` yet.
+> macOS only component
+
+The `swiftDialog` executable can be generated from a macOS host by running:
+```sh
+make swift-dialog-app-tar-gz version=2.2.1 build=4591 out-path=.
+```
+```sh
+fleetctl updates add --target /path/to/macos/swiftDialog.app.tar.gz --platform macos --name swiftDialog --version 2.2.1 -t edge
+```
+
+#### Releasing nudge to `stable`
+
+> `releaser.sh` doesn't support `nudge` yet.
+> macOS only component
+
+The `nudge` executable can be generated from a macOS host by running:
+```sh
+make nudge-app-tar-gz version=1.1.10.81462 out-path=.
+```
+```sh
+fleetctl updates add --target /path/to/macos/nudge.app.tar.gz --platform macos --name nudge --version 1.1.10.81462 -t edge
+```
+
```

Comment:
@georgekarrv @roperzh Heads up that this script is available if/when y'all want to update to include `swiftDialog` and `nudge`.

Reply:
+1, when you are about to release those components, please add the commands to `releaser.sh`and release using the script :)